### PR TITLE
chore(spanner): modify benchwrapper's proto to use Singers/Albums

### PR DESCRIPTION
**Pull request by zoercai**

### CODE REVIEW

File: spanner/go.mod [hunk 1]
```diff
@@ -12,5 +12,6 @@ require (
 	google.golang.org/api v0.54.0
 	google.golang.org/genproto v0.0.0-20210825212027-de86158e7fda
 	google.golang.org/grpc v1.40.0
+	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.1.0 // indirect
```
Comment (hengfengli) [inline spanner/go.mod:15]: I don't think we should add this dependency. We probably should remove the changes in go.mod and go.sum.

File: spanner/internal/benchwrapper/main.go [hunk 2]
```diff
@@ -80,8 +81,9 @@ func (s *server) Read(ctx context.Context, req *pb.ReadQuery) (*pb.EmptyResponse
 
 func (s *server) Insert(ctx context.Context, req *pb.InsertQuery) (*pb.EmptyResponse, error) {
 	var muts []*spanner.Mutation
-	for _, i := range req.Users {
-		muts = append(muts, spanner.Insert("sometable", []string{"name", "age"}, []interface{}{i.Name, i.Age}))
+	fmt.Print(req.Singers)
```
Comment (hengfengli) [inline spanner/internal/benchwrapper/main.go:84]: is this for debugging?

File: spanner/internal/benchwrapper/proto/spanner.proto [hunk 3]
```diff
@@ -15,31 +15,42 @@
...
-message UpdateQuery{
+message UpdateQuery {
   // The queries to use in the update call.
-  repeated string Queries = 1;
+  repeated string queries = 1;
 }
 
-message EmptyResponse{
-}
+// TODO(deklerk): Replace with Google's canonical Empty.
```
Comment (hengfengli) [inline spanner/internal/benchwrapper/proto/spanner.proto:50]: why do we add this TODO as deklerk is not working on this any more?

File: spanner/internal/benchwrapper/main.go (hunk 2 above)
Comment (zoercai) [reply spanner/internal/benchwrapper/main.go:84]: ah yep thanks for spotting - removed!

File: spanner/go.mod (hunk 1 above)
Comment (zoercai) [reply spanner/go.mod:15]: I've removed both

File: spanner/internal/benchwrapper/proto/spanner.proto (hunk 3 above)
Comment (zoercai) [reply spanner/internal/benchwrapper/proto/spanner.proto:50]: Oh I copied this proto from the one in google3, which had this todo, I've removed it now.

**Review by hengfengli — APPROVED** [review]
LGTM.

**Comment by zoercai** [comment]
@codyoss do you know why the kokoro CI build is failing here? There doesn't seem to be anything in the logs that show what's causing the failure.

**Comment by codyoss** [comment]
@zoercai It looks like you need to run `goimports -w .` in the directory of the updated proto. Linting error

**Pull request merged**